Backends/CLX/medium.lisp

```diff
@@ -406,10 +406,19 @@ translated, so they begin at different position than [0,0])."))
       (push #'(lambda () (xlib:free-pixmap mm)) ^cleanup)
       mm)))
 
+;;; For more details, see also: https://github.com/sharplispers/clx/pull/146
+(defun cached-drawable-depth (drawable)
+  "The purpose of this is to reduce local network traffic for the case of many
+calls to compute-rgb-image, for example when drawing a pattern."
+  (symbol-macrolet ((plist (xlib:drawable-plist drawable)))
```

Comment:
there is no real need for symbol-macrolet (form is short and used only twice). Please remove it.

Reply:
Okay -- I'll note for the record though that in my opinion, it is much more readable with this macrolet.